backend/src/middleware/slugify/uniqueSlug.spec.js

```diff
@@ -19,3 +19,17 @@ describe('uniqueSlug', () => {
     expect(uniqueSlug(string, isUnique)).resolves.toEqual('anonymous')
   })
 })
+
+describe('Slug is transliterated correctly', () => {
```

Comment:
Please have only one top level describe block. You can easily move the second one inside the first one.